### PR TITLE
Added RaspberryPI support

### PR DESCRIPTION
**Pull request by PAException**
This pull request includes:

- [ ] breaking changes
- [x] no breaking changes

### Changes made to the repository:

Added RaspberryPI support

### Documentation of test results:

Tested CloudNet on RaspberryPI, located error --> de.dytanic.cloudnet.console.JLine2Console.<init> AnsiConsole#systemInstall
Now the CloudNet checks if the OS is a raspberryPi and just if this is the case the command is executed.

### Related issues/discussions:

https://discordapp.com/channels/325362837184577536/553304283857289268/765208993030209556

### CODE REVIEW

File: cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java [hunk 1]
```diff
@@ -60,6 +60,20 @@ public JLine3Console() throws Exception {
         this.consoleReadThread.start();
     }
 
+    private void checkAnsiConsole() {
+        String[] OSAnsiConsoleNotSupported = new String[]{"raspi"};
```
Comment (einrobin) [inline cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java]: Variables are named in the lowerCamelCase

Comment (einrobin) [inline cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java]: Is there any specific reason for this array?

File: cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java [hunk 2]
```diff
@@ -47,7 +47,7 @@
 
     public JLine3Console() throws Exception {
         System.setProperty("library.jansi.version", "CloudNET");
-        AnsiConsole.systemInstall();
+        checkAnsiConsole();
```
Comment (einrobin) [inline cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java]: this.

File: cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java [hunk 3]
```diff
@@ -60,6 +60,20 @@ public JLine3Console() throws Exception {
         this.consoleReadThread.start();
     }
 
+    private void checkAnsiConsole() {
+        String[] OSAnsiConsoleNotSupported = new String[]{"raspi"};
+        String currentOS = System.getProperty("os.version");
+        boolean ansiConsoleSupported = true;
+        for (String os : OSAnsiConsoleNotSupported) {
+            if (currentOS.contains(os)) {
+                ansiConsoleSupported = false;
```
Comment (einrobin) [inline cloudnet/src/main/java/de/dytanic/cloudnet/console/JLine3Console.java]: Just execute systemInstall directly without this additional boolean

**Review by einrobin — CHANGES_REQUESTED** [review]

**Comment by PAException** [comment]
@derrop Revised and just checked for raspi now

**Review by einrobin — APPROVED** [review]

**Review by juliarn — APPROVED** [review]

**Pull request merged**